tests/src/Loader/binding/tracing/BinderTracingTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+using Assert = Xunit.Assert;
+
+namespace BinderTracingTests
+{
+    class BinderTracingTest
+    {
+        public static void PlatformAssembly_DefaultALC()
+        {
+            Console.WriteLine($"Running {nameof(PlatformAssembly_DefaultALC)}...");
+            using (var listener = new BinderEventListener())
+            {
+                string assemblyName = "System.Xml";
+                Assembly asm = Assembly.Load(assemblyName);
+
+                BindEvent[] events = listener.WaitAndGetEventsForAssembly(assemblyName);
+                Assert.True(events.Length == 1, $"Bind event count for {assemblyName} - expected: 1, actual: {events.Length}");
+                BindEvent bindEvent = events[0];
+                Assert.True(bindEvent.Success, $"Expected bind for {assemblyName} to succeed");
+            }
+        }
+
+        public static void NonExistentAssembly_DefaultALC()
```

Comment:
We have just two tests for this at the moment, so a little bit of copying/pasting is OK, but we might want to make this a table-based test instead.

Reply:
Yeah, we should be able to pull out some common code as we add more cases; I didn't try to do anything around that preemptively. We'll also likely have some shared parts for things like not polluting the default ALC for other tests (maybe we could use the `RemoteExecutor` - although nothing else in coreclr seems to use it right now)